examples/ios/swift/ListSwiftUI/Views/ContentView.swift

```diff
@@ -0,0 +1,103 @@
+import SwiftUI
+import RealmSwift
+
+struct RecipeRow: View {
+    var recipe: Recipe
```

Comment:
I believe that this line of code could cause a bug. If the recipe changes (as in a property of it changes), but the pointer of the object doesn't change, then this view won't update.

See my comment here: https://github.com/realm/realm-cocoa/issues/6162#issuecomment-557234370

This is a pretty insane way that SwiftUI behaves...it seems like a bug in SwiftUI to me but Apple claims that it behaves as expected.